docs/site/Model.md

```diff
@@ -851,6 +851,27 @@ class TestModel {
 }
 ```
 
+To define a nested array property, you must provide the `jsonSchema` field to
```

Comment:
I just realized that our docs https://loopback.io/doc/en/lb4/Model.html barely mentions `jsonSchema` and what it does and why it is related to our model definition. Do we name it `jsonSchema` because the model definition is translated to a JSON?

Reply:
@agnes512 good catch I didn't realize there is hidden content on this page. 
But the hidden div ends at line 776, so it doesn't affect the doc of array decorator :)

Reply:
But I don't see that part under https://loopback.io/doc/en/lb4/Model.html#data-mapping-properties.

Reply:
@agnes512 ah, fixes it in PR https://github.com/strongloop/loopback-next/pull/5568